packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -209,7 +209,9 @@ export default async args => {
             commonjs({
                 extensions: config.extensions,
                 transformMixedEsModules: true,
-                requireReturnsDefault: true,
+                requireReturnsDefault: id => {
+                    return !id.includes("react-lifecycles-compat");
+                },
```

Comment:
This is necessary for similar reasons when I tried to fix `react-image-lightbox`. Both use `react-modal`, which somewhere uses `react-lifecycles-compat` which needs to return a `false` value here otherwise it own't compile.